### PR TITLE
[WFLY-19671] Remove 'galleon content only' dependencies from the list…

### PR DESCRIPTION
**Pull request by bstansberry**
… of feature pack deps

https://issues.redhat.com/browse/WFLY-19671

### CODE REVIEW

File: ee-feature-pack/galleon-feature-pack/pom.xml [hunk 1]
```diff
@@ -237,12 +237,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- WFLY-19671. DO NOT include 'galleon content only' dependencies
```
Comment (jfdenise) [inline ee-feature-pack/galleon-feature-pack/pom.xml:240]: @bstansberry , I can see some remaining usages of wildfly-core-feature-pack-galleon-common (ban-transitive-deps and update-licenses-xml). BTW, perhaps cleaning wildfly-core-feature-pack-galleon-pruned too?

Comment (jfdenise) [reply ee-feature-pack/galleon-feature-pack/pom.xml:240]: @bstansberry , another point, just to be sure... Looking at https://github.com/wildfly/wildfly-core/tree/main/core-feature-pack/galleon-common/src/main/resources/packages/org.wildfly.openssl/pm/wildfly and https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/galleon-common/src/main/resources/packages/core.misc.common/pm/wildfly/tasks.xml we can see artifacts that require a version.

These artifacts are resolved thanks to the dependency in https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/pom.xml so we are good. 

Perhaps in wildfly-core we should add a note to not add dependencies to galleon-common, add them to common?

Comment (bstansberry) [reply ee-feature-pack/galleon-feature-pack/pom.xml:240]: @jfdenise re your second comment, I filed the much broader scoped https://issues.redhat.com/browse/WFCORE-6755. Thanks for pointing this out. I've long had the urge to get core-feature-pack reorganized but never had a reason to push it.

Comment (bstansberry) [reply ee-feature-pack/galleon-feature-pack/pom.xml:240]: Re your first comment -- good catch. I just pushed an update cleaning those out.

**Review by jfdenise — APPROVED** [review]

**Comment by bstansberry** [comment]
BTW in my earlier response to your review I posted a link to some random JIRA, not the one I filed. The right link is https://issues.redhat.com/browse/WFCORE-6961.

**Pull request merged**